tests/test_authentication.py

```diff
@@ -264,5 +264,5 @@ def test_login_required_with_return_app_sets_user_attributes_with_valid_token(
         assert mock_request.status_code == 200
         assert (
             mock_request.json["logout_url"]
-            == "https://authenticator/sessions/sign-out?return_app=post-award-frontend"
+            == "https://authenticator/sessions/sign-out?return_app=post-award-frontend&return_path=%2Fmock_login_requested_return_app_route"  # noqa: E501
```

Comment:
Should we add a test case for a path that includes some query params? need to make sure those also get encoded correctly I guess.

Reply:
@samuelhwilliams Good spot, it wasn't retaining the query string before, I've updated it now in 0dbadfd